project/lipper1110/README.md

```diff
@@ -0,0 +1,83 @@
+# Quantitative Trading System
+
+## Basic Information
+
+Quantitative trading is executed by defining mathematical models or algorithmic strategies that can be used to evaluate the profit of the strategy through historical data over time and events. It eliminates the emotional and cognitive biases that can affect people's trading decisions and also avoids potential costs and gain more profit by the fast and precise orders.
+
+
+## Problem to Solve
+Quantitative trading involves the following components, which we will carry out based on three time period strategies :
+* Strategy selection
+    * scalping trading : **A rapid, very short term trading style**
+        * Depends on its ability to perform trading actions at critical points in time in an extremely latency-sensitive manner
+        * HFT (High Frequency Trading)
+    * swing trading : **A mid-term trading style**
+        * Involves taking trades that last a couple of days up to several months in order to profit from an anticipated price move.
+        * Technical analysis, analyzing price trend and patterns
+    * position trading : **A long-term trading style**
+        * Identify a trend and an investment that will benefit from it, then buy and hold the investment until the trend peaks
+        * Trend following
+* Strategy backtest
+    * Backtesting for different time periods according to different strategies
+    * Evaluate by Shape ratio, ROI and consider the potential costs, e.g. fees, slippage, etc
+* Strategy execution
+    * Need to quickly calculate the appropriate points of in and out
+    * Ordering at precise points to avoid high slippage and other potential costs
+
+Among them, **strategy execution** and **order** are particularly important for performance, since they directly affect the profit of the trading.
+
+However we cannot have a hardware platform and network environment like a financial institution (the physical location of the hardware platform is important for High Frequency Trading). So we focus on the implementation of algorithm for **strategy execution**
+
+## Prospective Users
+Financial institutions trader, hedge funds, retail investors, people who want to make money from the secondary market
+
+## System Architecture
+![](https://i.imgur.com/BK5h5LN.png)
+
+## API Description
+### Python
+* `crawler`
```

Comment:
The plan looks complete.  Crawling could take some time.  I do not know how much experience you have with crawling.  If not a lot, limiting the scope of data could be an option.

Reply:
Thanks, maybe I will crawl the market data for 5 years.